portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,6 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+function openLinkedIn(){
+    window.open("www.linkedin.com/in/elijahrrb");
+}
```

Comment:
Nit: make sure your files end with a newline. A lot of different utilities expect one.